smac/intensification/intensification.py

```diff
@@ -270,11 +270,11 @@ def intensify(self, challengers, incumbent, run_history, aggregate_func,
                     # challenger is not worse, continue
                     N = 2 * N
 
-            if chall_indx >= 1 and num_run > self.run_limit:
+            if chall_indx > 1 and num_run > self.run_limit:
```

Comment:
We should consider counting the challenger index differently to make sure that we run two new configurations. Currently, it can happen that a challenger is identical to the incumbent or that we pick a challenger for which all runs are completed. Doing so would counteract the goal of this change.